File: day1/silverStar.py
with open('day1/input.txt', 'r') as file:
    left_list = []
    right_list = []
    for line in file:
        if line.strip():
            left_num, right_num = map(int, line.strip().split())
            left_list.append(left_num)
            right_list.append(right_num)

left_list.sort()
right_list.sort()

total_distance = sum(abs(l - r) for l, r in zip(left_list, right_list))

print(total_distance)

File: R/day1/silverStar.R
data <- read.table("day1/input.txt", header = FALSE)

left_list <- data$V1
right_list <- data$V2

# Sort lists
sorted_left <- sort(left_list)
sorted_right <- sort(right_list)

# Compute differences
differences <- abs(sorted_left - sorted_right)

# Calculate total distance
total_distance <- sum(differences)

cat(total_distance, "\n")
